packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+a GraphQL transaction query.
+
+Instructions for the following programs are supported.
+
+- Address Lookup Table
+- BPF Loader
+- BPF Upgradeable Loader
+- Stake
+- SPL Associated Token
+- SPL Memo
+- SPL Token
+- System
+- Vote
+
+Note at this time Token 2022 extensions are not yet supported.
+
+Similar to accounts, transactions with encoded data are also supported.
+
+- `TransactionBase58`: A Solana transaction with base58 encoded data
+- `TransactionBase64`: A Solana transaction with base64 encoded data
+- `TransactionJson`: A Solana transaction with JSON data
+
+Specific instruction types can be used in the transaction's instructions. The
+default instruction if it cannot be parsed using `jsonParsed` is the JSON
+version dubbed `PartiallyDecodedInstruction`.
+
+```typescript
+const source = `
+    query myQuery($signature: String!, $commitment: Commitment) {
+        transaction(signature: $signature, commitment: $commitment) {
+            ... on TransactionJsonParsed {
+                transaction {
+                    message {
+                        ... on TransactionMessageParsed {
+                            accountKeys {
+                                pubkey
+                                signer
+                                source
+                                writable
+                            }
+                            instructions {
+                                ... on PartiallyDecodedInstruction {
+                                    accounts
+                                    data
+                                    programId
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const variableValues = {
+    signature: '63zkpxATgAwXRGFQZPDESTw2m4uZQ99sX338ibgKtTcgG6v34E3MSS3zckCwJHrimS71cvei6h1Bn1K1De53BNWC',
+    commitment: 'confirmed',
+};
+                                        
+const result = await rpcGraphQL.query(source, variableValues);
 
-⚠️ **In Development:** The API's query/schema structure may change as the API
-matures.
+expect(result).toMatchObject({
+    data: {
+        transaction: {
+            transaction: {
+                message: {
+                    accountKeys: expect.arrayContaining([
+                        {
+                            pubkey: expect.any(String),
+                            signer: expect.any(Boolean),
+                            source: expect.any(String),
+                            writable: expect.any(Boolean),
+                        },
+                    ]),
+                    instructions: expect.arrayContaining([{
+                        accounts: expect.any(Array), // string[]
+                        data: expect.any(String), // Base64 encoded instruction data
+                        programId: expect.any(String),
+                    }])
+                },
+            },
+        },
+    },
+});
```

Comment:
This is another one where I think showing an example output would be really nice!

Reply:
Added!